themes/OneDark.tmTheme

```diff
@@ -48,7 +50,7 @@
         <key>name</key>
         <string>Comment</string>
         <key>scope</key>
-        <string>comment, string.comment</string>
```

Comment:
curious where you found a comprehensive list of scopes aside from what TextMate documents?